### DISCUSSION TITLE
Very new to next.js

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#21598] [Help] [answered]
Hi everyone,

I am starting to learn programming with next and have a basic question.

Using the example from next.js tutorial page, I do run build and then run start, I am not able to see any changes appear in the browser after I edit the html file. All is fine when I use the development server

Is there anything special I need to do in order to make changes appear in the browser or is this expected behaviour?

**Comment by michrome** [comment, marked as answer]
It's possible you're running the static web server and not the development server.

You need to run `npm run dev` to see changes. See https://nextjs.org/learn/basics/create-nextjs-app/setup

**Reply by ghost** [reply]
Thanks. My goal is to build a project and then send a zip file of it to another PC that has node installed and running.

So when I npm run dev on my PC, all is fine. When I npm run build. I see a new folder ".next" is created. I now do npm run start and then I go to .next/server/pages and I assume these are the files I need to zip up. So I load index.html from this location in my browser and I see the page. I load index.html in notepad and make a change. This change is not auto updating in my browser and it is not updating after I refresh the page.

**Reply by michrome** [reply]
After the build, you need to run [`next export`](https://nextjs.org/docs/advanced-features/static-html-export) which will create an `out` directory by default. The contents of that directory can be served by a static web server e.g. Apache or NGINX. If you edit the HTML, you will have to reload the page in the browser to see the change.

**Reply by ghost** [reply]
Great, thank you. That's what I was looking for.

Last question, is there any way to do a next export and have the web page auto update when I make a change to the html page? Similar to how it behaves when I run the dev environment and edit the index.js file.

**Reply by michrome** [reply]
No problem. If you’ve built and exported then you have a static site which is optimised for serving from a web server—not editing—so you’re moving “off the rails” a little. That said, I think you could include something like https://www.browsersync.io/ in your HTML.

**Reply by ghost** [reply]
Thank you very much, that did the trick
